primer-io/primer-sdk-ios
# fix: DPS-292 fixes externalPayerInfo on Paypal

# Description

DPS-292

This change mirrors `externalPayerId`, `firstName` and `lastName` to their snake case counterparts. Note that the mirroring is necessary due to the fact that the API expects these values in snake case, but returns them in camelCase. See here:

<img width="1437" alt="Screenshot 2024-04-24 at 16 51 31" src="https://github.com/primer-io/primer-sdk-ios/assets/3179752/69b53687-6349-4417-b328-3837f18851ff">


# Manual Testing

Make a paypal payment, and check that External Payer ID, first name, and last name is set on the payment.

<img width="1574" alt="Screenshot 2024-04-24 at 16 50 40" src="https://github.com/primer-io/primer-sdk-ios/assets/3179752/9015be9e-16cd-40c1-874e-39045e8d7290">
